MarsQACompetitionTaskNUnit/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,144 @@
+﻿using MarsQACompetitionTaskNUnit.Assertions;
+using NUnit.Framework;
+
+namespace MarsQACompetitionTaskNUnit.Tests
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CertificationTest: BaseTest 
+    {
+        [SetUp]
+        public void SetUpCertification()
+        {           
+            certificationPageObject.NavigateToCertificationTab();
+            Thread.Sleep(1000);
+        }
+
+        [Test, Order(1), Description("This test create a new Certification record")]
+        public void TestCreateCertificationRecord()
+        {         
+            certificationPageObject.ClearCertification();
+            certificationPageObject.CreateCertificationRecord(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
+            AssertionHelpers.AssertToolTipMessage(certificationPageObject, certificationConfig[0].AssertionMessage);
+            bool recordPresent = certificationPageObject.IsCertificationRecordPresent(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
+            Assert.IsTrue(recordPresent);
+            certificationPageObject.ClearCertification();
```

Comment:
here you should only delete data this test is adding

Reply:
Delete specific data that I am adding